tests/openassetio/test/manager/test_main.py

```diff
@@ -52,7 +52,7 @@ def test_api_logging_goes_to_standard_out(self, a_passing_fixtures_file, monkeyp
         assert "[debug]: PluginSystem" in str(execute_cli(a_passing_fixtures_file).stdout)
 
     def test_unittest_output_written_to_stderr(self, a_passing_fixtures_file):
-        assert "---\\nRan" in str(execute_cli(a_passing_fixtures_file).stderr)
+        assert "Ran" in str(execute_cli(a_passing_fixtures_file).stderr)
```

Comment:
[consider][minor] Is it the platform-specific newline separator that tripped up here? If it's of value then [os.linesep](https://docs.python.org/3.9/library/os.html#os.linesep) may be useful.  Very minor though, happy to leave as is.

Reply:
Good idea, I left it as-is though as the whole complexity of also having to escape it felt like it was snowballing a little...